caffe2/python/examples/resnet50_trainer.py

```diff
@@ -641,7 +641,7 @@ def main():
     parser.add_argument('--dtype', default='float',
                         choices=['float', 'float16'],
                         help='Data type used for training')
-    parser.add_argument('--float16_compute', action='store_true',
+    parser.add_argument('--half_compute', action='store_true',
```

Comment:
Euh... BC?

Reply:
Yeah these python changes aren't supposed to be here. Thanks for catching it.